redfish-core/lib/service_root.hpp

```diff
@@ -26,30 +26,71 @@ namespace redfish
 inline void
     handleACFWindowActive(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
 {
+    // Redfish property ACFWindowActive = true when D-Bus property
+    // allow_unauth_upload is true (Redfish property AllowUnauthACFUpload).
     crow::connections::systemBus->async_method_call(
         [asyncResp](const boost::system::error_code ec,
-                    const std::variant<bool>& retVal) {
+                    const std::variant<bool>& allowed) {
             if (ec)
             {
-                BMCWEB_LOG_ERROR << "Failed to read ACFWindowActive property";
-                // Default value when panel app is unreachable.
-                asyncResp->res.jsonValue["Oem"]["IBM"]["ACFWindowActive"] =
-                    false;
+                BMCWEB_LOG_ERROR
+                    << "D-Bus response error reading allow_unauth_upload: "
+                    << ec;
+                messages::internalError(asyncResp->res);
                 return;
             }
-            const bool* isACFWindowActive = std::get_if<bool>(&retVal);
-            if (isACFWindowActive == nullptr)
+            const bool* allowUnauthACFUpload = std::get_if<bool>(&allowed);
+            if (allowUnauthACFUpload == nullptr)
             {
-                BMCWEB_LOG_ERROR << "nullptr for ACFWindowActive";
+                BMCWEB_LOG_ERROR << "nullptr for allow_unauth_upload";
                 messages::internalError(asyncResp->res);
                 return;
             }
-            asyncResp->res.jsonValue["Oem"]["IBM"]["ACFWindowActive"] =
-                *isACFWindowActive;
+            if (*allowUnauthACFUpload == true)
+            {
+                asyncResp->res.jsonValue["Oem"]["IBM"]["ACFWindowActive"] =
+                    *allowUnauthACFUpload;
+                return;
```

Comment:
Why return here if allowUnauthACFUpload  is true? 
I guess I would think ACFWindowActive and AllowUnauthACFUpload  are both always shown. 
If AllowUnauthACFUpload   is false, then ACFWindowActive  is false but I would still think in all cases, you would show both

Reply:
As I mentioned in the comment above, this URL only shows the Redfish property ACFWindowActive.
This value is computed from D-Bus property allow_unauth_upload and D-Bus property ACFWindowActive.
But D-Bus property allow_unauth_upload is not shown here, only under /redfish/v1/AccountService/Accounts/service property Oem.IBM.ACF